src/html.rs

```diff
@@ -100,6 +100,12 @@ where
                     escape_html(&mut self.writer, &text, false)?;
                     self.end_newline = text.ends_with('\n');
                 }
+                Code(text) => {
+                    self.write(b"<code>", false)?;
+                    escape_html(&mut self.writer, &text, false)?;
+                    self.end_newline = text.ends_with('\n');
```

Comment:
This is maybe a minor point, but since there's a `</code>` closing tag, maybe `end_newline` should be `false`?

Reply:
Good catch!